src/components/pages/InstructorHome/InstructorProfile.js

```diff
@@ -24,8 +24,8 @@ function InstructorProfile({ dummyData }) {
         </div>
       </Button>
       <Descriptions>
-        <Descriptions.Item label="Raing">{`${dummyData.instructors[1].rating}`}</Descriptions.Item>
-        <Descriptions.Item label="Bio">{`${dummyData.instructors[1].bio}`}</Descriptions.Item>
+        <Descriptions.Item label="Rating">{`${dummyData[1].rating}`}</Descriptions.Item>
```

Comment:
The instructors property was passed in through 'InstructorHome/index.js' so we don't need to reference it again since it is represented by 'dummyData' in this component.